cluster-autoscaler/debuggingsnapshot/debugging_snapshot.go

```diff
@@ -58,11 +61,12 @@ type DebuggingSnapshot interface {
 // Please add all new output fields in this struct. This is to make the data
 // encoding/decoding easier as the single object going into the decoder
 type DebuggingSnapshotImpl struct {
-	NodeInfo                      []*NodeInfo `json:"NodeList"`
-	UnscheduledPodsCanBeScheduled []*v1.Pod   `json:"UnscheduledPodsCanBeScheduled"`
-	Error                         string      `json:"Error,omitempty"`
-	StartTimestamp                time.Time   `json:"StartTimestamp"`
-	EndTimestamp                  time.Time   `json:"EndTimestamp"`
+	NodeInfo                      []*NodeInfo           `json:"NodeList"`
```

Comment:
nit: I know this is not introduced in this PR, but why is the field name different that the corresponding field in json? This seems needlessly confusing.
Also - can you make the name imply it's multiple nodes/nodeInfos? NodeList is good, but NodeInfo seems like it would be just a single object and not a list.

Reply:
Done. Changed Names.